webapi/src/main/scala/org/knora/webapi/responders/v1/ValueUtilV1.scala

```diff
@@ -513,6 +531,32 @@ class ValueUtilV1(private val settings: SettingsImpl) {
             referenceCount = predicates(OntologyConstants.KnoraBase.ValueHasRefCount).literals.head.toInt
         )
     }
+
+    /** Creates an attribute segment for the Salsah GUI from the given resource class.
+      * Example: if incunabula:book is given, the function returns "restypeid=incunabula:book".
```

Comment:
Should this be `incunabula:book` or `http://www.knora.org/ontology/incunabula#book`?